KubeArmor/common/common.go

```diff
@@ -456,3 +463,61 @@ func WriteToFile(val interface{}, destFile string) error {
 	}
 	return nil
 }
+
+// ParseURL with/without scheme and return host, port or error
+func ParseURL(address string) (string, string, error) {
+	var host string
+	port := "80"
+
+	addr, err := url.Parse(address)
+	if err != nil || addr.Host == "" {
+		// URL without scheme
+		u, repErr := url.ParseRequestURI("http://" + address)
+		if repErr != nil {
+			return "", "", fmt.Errorf("Error while parsing URL: %s", err)
+		}
+
+		addr = u
+	}
+
+	if addr.Port() != "" {
+		host = strings.Split(addr.Host, ":")[0]
```

Comment:
Why didn't we use host = addr.Hostname() here?

Reply:
Fixed. Thanks for the suggestion.